charts/lakefs/README.md

```diff
@@ -51,6 +51,26 @@ Sensitive information like `database_connection_string` is given through "secret
 You should give your Kubernetes nodes access to all S3 buckets you intend to use lakeFS with.
 If you can't provide such access, lakeFS can be configured to use an AWS key-pair to authenticate (part of the `lakefsConfig` YAML below).
 
+## Major Chart Upgrades
+### General GuidelinesBefore 
+Before performing a major upgrade, it is strongly recommended to perform these steps:
+* Commit all uncommitted data on branches
+* Create a snapshot of your database
```

Comment:
This are not reasonable requirements. 
Is this just for the KV upgrade? Why is this presented as a general major upgrade thing?

Reply:
I agree.
Removed it from the `General Guidelines` section (which was removed as it was empty) and moved these guidelines to the specific relevant upgrade.
Should the need arise, we can always add a `General Guidelines` section again